### PR TITLE
#2: Updated dependencies and documentation.

### LINKED ISSUE #2: Update dependencies of 1.0.0
# Situation

The JUnit dependency used in 1.0.0 has a know security issue.

# Acceptance Criteria

1. All versions updated to the latest
2. Version checker plugin-installed

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #2

### CODE REVIEW

File: doc/changes/changes_1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* Added `org.junit.jupiter:junit-jupiter-engine:5.6.2`
+* Added `org.junit.jupiter:junit-jupiter-params:5.6.2`
+* Added `org.sonatype.ossindex.maven:ossindex-maven-plugin:3.1.0`
+* Updated `maven-compiler-plugin:3.7.0` to `3.8.1`
+* Updated `maven-jar-plugin.3.1.0` to `3.2.0`
+* Updated `maven-javadoc-plugin.3.0.1` to `3.2.0`
+* Updated `maven-source-plugin.3.0.1` to `3.2.1`
+* Updated `maven-surefire-plugin.2.21.0` to `3.0.0-M5` 
+* Updated `org.codehaus.mojo:license-maven-plugin.1.16` to `1.20`
+* Updated `org.jacoco:jacoco-maven-plugin.0.8.1` to `0.8.6`
```
Comment (kaklakariada) [inline doc/changes/changes_1.0.1.md]: This mixes dependencies, test dependencies and build dependencies.
Maybe think about listing them separately.

File: pom.xml [hunk 2]
```diff
@@ -72,17 +72,17 @@
...
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>3.2.0</version>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.21.0</version>
+                <version>3.0.0-M5</version>
```
Comment (kaklakariada) [inline pom.xml:80]: Is it ok to use a non-release version?

File: pom.xml.versionsBackup [hunk 3]
```diff
@@ -0,0 +1,206 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (kaklakariada) [inline pom.xml.versionsBackup]: Is it necessary to add this file? Maybe ignore it?

File: doc/changes/changes_1.0.1.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# hamcrest-multiline-text-matcher 1.0.1, released 2020-12-19
+
+Codename: Update Frenzy
+
+## Summary 
+
+In version 1.0.1 we ported all tests from JUnit 4 to JUnit 5. We also updated all other dependencies.
```
Comment (kaklakariada) [inline doc/changes/changes_1.0.1.md]: Breaking change: upgrade from java 8 to 11

File: pom.xml [hunk 5]
```diff
@@ -8,13 +8,13 @@
     <description>This project provides a Hamcrest matcher that gives you better diagnostics for multi-line text in case of mismatches.</description>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```
Comment (kaklakariada) [inline pom.xml:11]: Remove Java 8 from travis ci config

**Review by kaklakariada — CHANGES_REQUESTED** [review]
Thanks for your PR, only minor findings :-)

File: doc/changes/changes_1.0.1.md (hunk 4 above)
Comment (redcatbear) [reply doc/changes/changes_1.0.1.md]: Added. Renamed to 2.0.0.

File: doc/changes/changes_1.0.1.md (hunk 1 above)
Comment (redcatbear) [reply doc/changes/changes_1.0.1.md]: Separated.

File: pom.xml (hunk 2 above)
Comment (redcatbear) [reply pom.xml:80]: Normally not, but Surefire has only had milestone releases since more than two years. I hope they finish 3.0.0 soon.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**